### PR TITLE
TW 245 Refresh  /collaborating-in-postman/version-control-for-collections

### PR DESCRIPTION
**Pull request by HeatherStenson**
- Updates screenshots
- Reorganizes content 
- Reformats instructions into numbered lists

(Follow-up work is necessary for the [Resolving conflicts](https://learning.postman.com/docs/collaborating-in-postman/version-control-for-collections/#resolving-conflicts) section and to document the use of **Refresh**, new ticket is TW 371)

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 1]
```diff
@@ -28,275 +29,341 @@ contextual_links:
...
+    * [Merging changes](#merging-changes)
 * [Resolving conflicts](#resolving-conflicts)
 
-## Forking a collection
-
-To fork a collection in Postman, select the collection in the __Collections__ sidebar. Select <img alt="Three dots icon" src="https://assets.postman.com/postman-docs/icon-three-dots-v9.jpg" width="18px" style="vertical-align:middle;margin-bottom:5px">  to view more actions, then select __Create a fork__. To provide a uniform forking experience, you can create a fork in a public workspace in three steps — log in to Postman, fill up fork details, and enable your public profile.
+## Forking Postman elements
 
-<img src="https://assets.postman.com/postman-docs/collection-create-a-fork-v9.1.jpg" alt="Create fork selected in menu" width="300px"/>
+A _fork_ is a new instance of an element that you can modify without making any changes to the parent element. In Postman, you can fork [collections](#forking-a-collection) and [environments](#forking-an-environment). Forking also enables you to contribute to a collection or an environment without having Editor access to that element.
```
Comment (jkonrath-postman) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:55]: Are we bolding access levels, like **Editor**?

**Review by jkonrath-postman — APPROVED** [review]
LGTM! I made a few little comments, but otherwise looks great.

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 2]
```diff
@@ -28,275 +29,341 @@ contextual_links:
...
+You can edit a pull request's details before approving it, or you can decline it.
 
-## Pulling updates
+<img alt="Refresh, edit, or decline a pull request" src="https://assets.postman.com/postman-docs/refresh-edit-decline-pull-request.jpg" width="350px"/>
 
-You can keep your forked collections up to date with any changes in the parent, for example if another team member has merged changes into the parent collection.
+To edit the pull request details:
 
-To compare your fork to its parent, choose __Merge changes__ in the forked collection in Postman.
+* Select the three dots <img alt="Three dots icon" src="https://assets.postman.com/postman-docs/icon-three-dots-v9.jpg" width="18px" style="vertical-align:middle;margin-bottom:5px"> from the pull request's header bar.
```
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:283]: The header is the very top bar of the Postman window. Maybe change this to something like:
Select the three dots at the upper right and select **Edit**.

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 3]
```diff
@@ -28,275 +29,341 @@ contextual_links:
...
+* Select **Edit**.
+* Make your changes to the pull request's title, description, and list of reviewers.
```
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:285]:
```suggestion
* Make any changes to the pull request's title, description, and list of reviewers.
```

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 4]
```diff
@@ -28,275 +29,341 @@ contextual_links:
...
+* Select **Save Changes**.
 
-<img src="https://assets.postman.com/postman-docs/collection-fork-merge-changes-v9.1.jpg" alt="Merge Fork" width="300px"/>
+If you do not want to merge the pull request into the parent collection, you can decline it. Declined pull requests cannot be reopened, so if you want to request edits or offer feedback, [add a comment](#adding-comments) instead. To decline the pull request:
 
-Postman will warn you before you attempt to merge a fork whose parent has changed since you last updated it. Select __Pull Changes__ to update your fork with the changes in the parent collection.
+* Select the three dots <img alt="Three dots icon" src="https://assets.postman.com/postman-docs/icon-three-dots-v9.jpg" width="18px" style="vertical-align:middle;margin-bottom:5px"> from the pull request's header bar.
```
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:290]: Same comment as above. Maybe change to:
Select the three dots at the upper right and select **Decline**.

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 5]
```diff
@@ -28,275 +29,341 @@ contextual_links:
...
 
-To view the differences between the fork (the _source_) and the parent fork (the _destination_), select __Changes__ tab.
+You can keep your forked collections up to date with any changes in the parent, for example if another team member has merged changes into the parent collection.
+
+1. Open the pull request. Postman will warn you that the parent collection has changed since you last updated it.
+1. Select **Pull Changes** to update your fork with the changes in the parent collection.
+
+![Pull recent changes](https://assets.postman.com/postman-docs/pr-pull-changes-v9.12.jpg)
+
+### Merging changes
```
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:317]: Add an intro sentence after this heading. Something like:
Merging updates the parent collection with changes from the pull request or forked collection.

**Review by mark-anthony-postman — APPROVED** [review]
Wow, this is all super good! 🌈  A huge improvement in terms of organization and structure, and all the new content is really clear and nicely written 🌟 Made some minor editorial comments you can use as you will 🙂

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 6]
```diff
@@ -28,275 +29,341 @@ contextual_links:
...
 
-Select the bell icon <img alt="Changelog icon" src="https://assets.postman.com/postman-docs/icon-notification-bell-v9.jpg" width="18px" style="vertical-align:middle;margin-bottom:5px"> in the top right corner of Postman to view the notification. The popup will indicate further information about the change that was made to the pull request.
+1. Navigate to the collection and select the **Pull Requests** icon <img alt="Pull request icon" src="https://assets.postman.com/postman-docs/icon-pull-request.jpg" width="16px" style="vertical-align:middle;margin-bottom:5px"> on the context menu.
 
-<img alt="Notification Pull Request Watching" src="https://assets.postman.com/postman-docs/pull-request-watch-notification-v8.jpg" width="500px"/>
+<img src="https://assets.postman.com/postman-docs/open-pull-request-list-v9.12.jpg" alt="Pull request list" width="350px"/>
 
-> If you created the pull request and modified it from the same account, you will not receive any notifications for changes made.
+Each item shows the pull request's status, which will be `OPEN` for any that have not been merged or declined. Click on a pull request's name to open it.
```
[the remark this answers is not in the document]
Comment (HeatherStenson) [reply src/pages/docs/collaborating-in-postman/version-control-for-collections.md:248]: Eep! 🙈  I'll shake this habit soon, I hope

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md (hunk 1 above)
Comment (jkonrath-postman) [reply src/pages/docs/collaborating-in-postman/version-control-for-collections.md:55]: This sounds like the perfect thing for a style guide committee to discuss. :) Go ahead and leave this for now, and we'll talk offline.

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md (hunk 5 above)
Comment (HeatherStenson) [reply src/pages/docs/collaborating-in-postman/version-control-for-collections.md:318]: Good call! Updated in [b2a4396](https://github.com/postmanlabs/postman-docs/pull/3882/commits/b2a4396465f57067827fe0691b44aca5149103a5)

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md (hunk 2 above)
Comment (HeatherStenson) [reply src/pages/docs/collaborating-in-postman/version-control-for-collections.md]: Thanks! Updated in [b2a4396](https://github.com/postmanlabs/postman-docs/pull/3882/commits/b2a4396465f57067827fe0691b44aca5149103a5)

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 7]
```diff
@@ -28,275 +29,341 @@ contextual_links:
...
-If your pull request has no conflicts, you can go ahead and open it for review. Enter a title and description, and select up to three reviewers from the dropdown list. Reviewers will need edit access to the collection in order to merge your changes. Select __Create Pull Request__.
+    <img src="https://assets.postman.com/postman-docs/make-source-collection-public-v9.jpg" alt="Make the source collection public" width="400px"/>
 
-<img alt="Create Pull Request" src="https://assets.postman.com/postman-docs/pull-request-overview-v2.jpg"/>
+1. After you move the collection to a public workspace, proceed with the workflow in [Creating pull requests](#creating-pull-requests).
 
-Reviewers can [comment on your pull request or decide to merge](#reviewing-pull-requests) your changes into the parent collection.
+Once you create the pull request, you will get a notification that it has been **Shared to public workspace**.
 
 ### Pull request settings
```
[the remark this answers is not in the document]
Comment (HeatherStenson) [reply src/pages/docs/collaborating-in-postman/version-control-for-collections.md:172]: Good call! Updated in [b2a4396](https://github.com/postmanlabs/postman-docs/pull/3882/commits/b2a4396465f57067827fe0691b44aca5149103a5)

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 8]
```diff
@@ -28,275 +29,341 @@ contextual_links:
...
+1. Select **Fork Collection**.
 
-[![Fork Information](https://assets.postman.com/postman-docs/fork-information-count.jpg)](https://assets.postman.com/postman-docs/fork-information-count.jpg)
+    <img src="https://assets.postman.com/postman-docs/fork-collection-v9.1.jpg" alt="Create fork tab" width="400px"/>
 
-> You can select a user under __Forked by__ to view their public profile.
+Your fork will be created in the selected workspace.
 
-### Forking to send requests
+If there are any [mocks](/docs/designing-and-developing-your-api/mocking-data/setting-up-mock/) or [monitors](/docs/monitoring-your-api/intro-monitors/) associated with a collection, they will not be available with the forked collection. You will need to create mocks and monitors specifically for the fork if you need them.
```
[the remark this answers is not in the document]
Comment (HeatherStenson) [reply src/pages/docs/collaborating-in-postman/version-control-for-collections.md]: Yes, good thinking! Updated in [b2a4396](https://github.com/postmanlabs/postman-docs/pull/3882/commits/b2a4396465f57067827fe0691b44aca5149103a5)

**Pull request merged**